src/components/Header.svelte

```diff
@@ -1,8 +1,16 @@
 <script lang="ts">
   import { goto } from '$app/navigation'
-  import { sessionStore } from '../stores'
+  import { sessionStore, theme } from '../stores'
+  import { Theme, storeTheme } from '$lib/theme/index'
```

Comment:
Could we import `Theme` as a type to make that explicit?

```suggestion
  import { storeTheme, type Theme } from '$lib/theme/index'
```